### PR TITLE
chore: fix homepage link in neuron-ui/package.json

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-ui/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "email": "dev@nervos.org",
     "url": "https://github.com/nervosnetwork/neuron"
   },
-  "homepage": "./",
+  "homepage": "https://www.nervos.org",
```
Comment (Keith-CY) [inline packages/neuron-ui/package.json]: This field sets the root of the web app, not the homepage of the project.

Comment (classicalliu) [reply packages/neuron-ui/package.json]: Sounds we can remove this line, no url for desktop app.

Comment (ashchan) [reply packages/neuron-ui/package.json]: True

Comment (Keith-CY) [reply packages/neuron-ui/package.json]: Not sure, the CRA fetch static files from `/` by default. And the Electron serves static files with the file system. Namely, the assets will be fetched from `file:///styles/xxx.css` which leads to errors. No verified. Please have a try.

**Pull request closed without merging**